### ISSUE TITLE
WITH clause for indexes

### ISSUE THREAD

**Issue by JerrySievert** [oguimbal/pgsql-ast-parser#66]
Hi there!

I found another portion of the index definition that doesn't seem to be supported.  Just hoping it's an easy addition for you.

here's an example:
```
CREATE INDEX foo ON bar USING btree (bar) WITH (fillfactor='70')
```

Thanks much!

**Comment by oguimbal** [comment]
Fixed in `pgsql-ast-parser@9.3.0`

Sorry for the delay !

**Comment by JerrySievert** [comment]
awesome, thanks so much!
